prow/kube/prowjob.go

```diff
@@ -105,6 +106,13 @@ func (j *ProwJob) Complete() bool {
 	return !j.Status.CompletionTime.IsZero()
 }
 
+func (j *ProwJob) ClusterAlias() string {
+	if j.Spec.Cluster == "" {
```

Comment:
When will this be empty? It looks like in `pjutil` you're setting it to `kube.DefaultClusterAlias` when its turned into a PJ spec.

Reply:
It will be empty for existing ProwJobs already in etcd.